.eslintrc.js

```diff
@@ -25,8 +40,10 @@ module.exports = {
       "components": ["Link"],
       "specialLink": ["to"],
     }],
+    "react/jsx-filename-extension": ["error", { "extensions": [".tsx", ".jsx"] }],
     "react/no-array-index-key": 1,
     "react/require-default-props": 0,
+    "import/extensions": 0,
```

Comment:
it looks like I lost the battle with setting up `eslint-plugin-import` with typescript: `Missing file extension "tsx" for "./SocialLinks"     import/extensions`

Reply:
IMO that's fine